packages/use-contractkit/src/contract-kit-provider.tsx

```diff
@@ -49,34 +58,49 @@ const initialState = {
     icon: 'https://celo.org/favicon.ico',
   },
   network: Mainnet,
-  networks: [],
+  networks: DEFAULT_NETWORKS,
   pendingActionCount: 0,
   address: null,
   connectionCallback: null,
 };
 
-export const ContractKitContext = createContext<ContractKitContextInterface>([
-  initialState,
-  () => undefined,
-]);
+export const [useContractKitContext, ContextProvider] =
+  createContractKitContext();
+
+function createContractKitContext() {
+  const contractKitContext = React.createContext<
+    ContractKitContextInterface | undefined
+  >(undefined);
+  function useCtx() {
```

Comment:
Nit: recommend using arrow functions for defining inlined functions as in some cases (though not this one) it can help avoid `this` binding issues